### ISSUE TITLE
Import from npm modules

### ISSUE THREAD

**Issue by anarasimhan** [ardatan/graphql-import#256]
Is it possible to load schema from other npm modules? All the examples I have seen so far is importing graphql files from the same application or module. 

Something like 

```
# import Post from 'posts.graphql'
# import Author from 'author-module/schema.graphql'
```

**Comment by taras** [comment]
We need to be able to do the same thing. We publish graphql schemas a npm packages. We would like to be able to import these into a single graphql package. One difference with our setup is that our packages have schema property in package.json that we use like an entry point. It's like `main` for JavaScript.

**Comment by ardatan** [comment]
Available in 1.0.0!

**Comment by taras** [comment]
Thank you!
